src/pages/Admin/AdminUpdateClass/AddNewClass.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useForm } from 'react-hook-form';
+import { useParams } from 'react-router-dom';
+import { Input, InputField } from '../../../components/Input';
+import { ErrorMessageType } from '../../../types';
+import {
+  adminCreateClass,
+  AdminCreateClassProps,
+} from '../../../api/getAdminClasses';
+
+import {
+  FormErrorMessages,
+  StyledErrorMessage,
+} from '../../../components/ErrorMessage';
+import { StyledSuccessMessage } from '../../../components/SuccessMessage';
+
+const StyledForm = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+`;
+
+const classValidationRules = {
+  name: {
+    required: 'Class Title is required',
+  },
+  subject: {
+    required: 'Class Description is required',
+  },
+  date: {
+    required: 'Date is required',
+  },
+};
```

Comment:
I think this can be moved to `utilities/validation.tsx`

Reply:
Yes, I can do that, I saw that you had created a utilities file in your recent PR, I will move it there. :)